Wikipedia/UI-V5/WMFSaveableTitleCollectionViewCell.h

```diff
@@ -0,0 +1,34 @@
+
+#import <SSDataSources/SSDataSources.h>
+
+@class MWKTitle;
+@class MWKSavedPageList;
+@class MWKImage;
+
+/**
+ *  Base class for cells which represent a title and have save button.
+ *
+ *  Will also be given the "card" style by applying a white background to its
+ *  @c contentView as well as a small shadow.
+ */
+@interface WMFSaveableTitleCollectionViewCell : SSBaseCollectionCell
```

Comment:
I like the current name as it gives you some clue as to what's special about that class (i.e. it's a save-able cell).  However, given that (I think) we want all our cells to be save-able, we could rename this to something like `WMFBaseCollectionViewCell`.